package.json

```diff
@@ -82,7 +82,7 @@
     "@fastify/type-provider-typebox": "^3.3.0",
     "@hyperswarm/secret-stream": "^6.1.2",
     "@mapeo/crypto": "^1.0.0-alpha.4",
-    "@mapeo/schema": "^3.0.0-next.5",
+    "@mapeo/schema": "file:../mapeo-schema/mapeo-schema-3.0.0-next.5.tgz",
```

Comment:
Is this change intentional?

Reply:
Yeah, until we merge and publish @mapeo/schema - a reminder not to merge this yet - tests won't pass